pkg/domain/entities/types/containers.go

```diff
@@ -51,12 +51,16 @@ type ContainerUpdateOptions struct {
 	// - DevicesLimits to Limit device
 	// - RestartPolicy to change restart policy
 	// - RestartRetries to change restart retries
+	// - Env to change the environment variables.
+	// - UntsetEnv to unset the environment variables.
 	Specgen                         *specgen.SpecGenerator
 	Resources                       *specs.LinuxResources
 	DevicesLimits                   *define.UpdateContainerDevicesLimits
 	ChangedHealthCheckConfiguration *define.UpdateHealthCheckConfig
 	RestartPolicy                   *string
 	RestartRetries                  *uint
+	Env                             *[]string
+	UnsetEnv                        *[]string
```

Comment:
I don't see a particular need for these to be pointers, the slice already can be nil which would mean nothing was explicitly set

Reply:
Technically len(0) is different from nil and if this was a plain setter that would matter - a len-0 slice would be no environment variables, nil would be "unset by user, use defaults" - but in this case it doesn't matter